backend/src/migrations/20230826220746_add_bsa_badges_table.ts

```diff
@@ -0,0 +1,43 @@
+import { type Knex } from 'knex';
+
+const uuid = 'uuid_generate_v4()';
+const constraintName = 'bsa_badges_pkey';
+
+const TABLE_NAME = 'bsa_badges';
+
+const ColumnName = {
+    ID: 'id',
+    TYPE: 'type',
+    NAME: 'name',
+    MAX_SCORE: 'max_score',
+    CREATED_AT: 'created_at',
+    UPDATED_AT: 'updated_at',
+};
+
+async function up(knex: Knex): Promise<void> {
+    return knex.schema.createTable(TABLE_NAME, (table) => {
+        table
```

Comment:
When you use uuid function first of all you need to create extension if not exists, because in other db this migration can fall down without the necessary extension. Every migration need to has this line when you use uuid

```suggestion
async function up(knex: Knex): Promise<void> {
    await knex.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";');

    return knex.schema.createTable(TABLE_NAME, (table) => {
        table
```